examples/with-loading/components/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Head from 'next/head'
+import MyLink from './MyLink'
+
+export default () => (
+  <div style={{ marginBottom: 20 }}>
+    <Head>
+      {/* Import CSS for nprogress */}
+      <link rel='stylesheet' type='text/css' href='http://ricostacruz.com/nprogress/nprogress.css' />
```

Comment:
maybe make a copy of this into `/static` instead? Since this is `http` and we don't want to spam his site